deploy.bash

```diff
@@ -278,7 +278,7 @@ else
 fi
 
 echo "8. Download JS... "
-if [[ $step =~ (^|,)6(,|$) || $step == "*" ]]; then
+if [[ $step =~ (^|,)8(,|$) || $step == "*" ]]; then
```

Comment:
This commit should be squashed with the previous one.

Reply:
Done.